app/models/project.rb

```diff
@@ -311,6 +311,10 @@ def target_grade=(value)
     self[:target_grade] = value
   end
 
+  def submitted_grade=(value)
```

Comment:
Do we really need this? Generally you should be able to do this without needing to actually write this code as ActiveRecord should handle it.